logstash-core/lib/logstash/pipeline.rb

```diff
@@ -37,6 +37,8 @@ module LogStash; class Pipeline
     :config_str,
     :original_settings
 
+  DEFAULT_OUTPUT_WORKERS = 1
+
   DEFAULT_SETTINGS = {
```

Comment:
Just to clarify, I think its confusing that we have dynamic settings and defaults in that hash. Defaults make more sense as constants IMHO.